docs/release-notes/8.18.asciidoc

```diff
@@ -0,0 +1,132 @@
+* Removes the critical services count from Entity Analytics dashboard summary panel ({kibana-pull}210827[#210827]).
+* Removes the prompt on the Entity Analytics dashboard that asks you to turn on the risk engine even though you have already done it ({kibana-pull}210430[#210430]).
+* Adds a filter to the entity definition schema so it can be used to further filter entity store data ({kibana-pull}208588[#208588]).
+* Improves the navigation and page descriptions for the Entity Store and Entity Risk Score pages ({kibana-pull}209130[#209130]).
+* Improves the confirmation message that appears when you update the configuration for a risk engine saved object ({kibana-pull}211372[#211372]).
+* Fixes a navigation issue with the host and user flyouts that prevented the flyout details from refreshing ({kibana-pull}209863[#209863]).
+* Ensures that you stay on your current page in the Rules table after editing or updating a rule ({kibana-pull}209537[#209537]).
+* Fixes a bug that caused the preview panel to incorrectly persist after you opened the session viewer preview ({kibana-pull}213455[#213455]).
+* Adds a "no data message" to the expanded event analyzer view in the alert details flyout when the event analyzer isn't turned on ({kibana-pull}211981[#211981]).
+* Fixes the order of the alert insights so they're now shown from low risk to critical risk({kibana-pull}212980[#212980]).
+* Fixes bugs that prevents cell action in the Alerts table from properly rendering in the event rendered view ({kibana-pull}212721[#212721]).
+* Fixes a bug that incorrectly concealed the the isolate host panel if you used the isolate host action from the alert preview ({kibana-pull}211853[#211853]).
+* Fixes a bug that prevented you from seeing alert assignee details from the Alerts table or the alert details flyout ({kibana-pull}211824[#211824]).
+* Fixes the width of the alerts table in rule preview ({kibana-pull}214028[#214028]).
+* Fixes a bug that prevented the rule creation form from properly validating EQL queries when you added filters to the query ({kibana-pull}212117[#212117]).
+* Makes 7.x alert indices compatible with Alerts table so you can access alerts in legacy indices ({kibana-pull}209936[#209936]).
+* Fixes a bug that didn't allow you to generate {esql} alerts from alert indices ({kibana-pull}208894[#208894]).
+* Surfaces shard failure details for failed EQL non-sequence queries on the rule details page and in the event log ({kibana-pull}207396[#207396]).
+* Fixes an {elastic-defend} bug to ensure the first event's timestamp is used as the timestamp for event aggregation.
+* Updates the way {elastic-defend} initially connects to {agent}, which significantly improves the speed of connection.
+* Fixes issues where uninstalling {elastic-defend] on Windows leaves files within {elastic-defend}'s directory that cannot be removed by administrators. These leftover files can prevent subsequent installs and upgrades.
```

Comment:
{elastic-defend] -> {elastic-defend}

Reply:
Addressed in [84b01ce](https://github.com/elastic/security-docs/pull/6519/commits/84b01cef0162353ff08baf9206adedacce200711).